src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,51 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_NUMBER = 10;
+    private K[] keys;
+    private V[] values;
+    private int size;
+
+    public StorageImpl() {
+        keys = (K[]) new Object[MAX_NUMBER];
+        values = (V[]) new Object[MAX_NUMBER];
+    }
+
     @Override
     public void put(K key, V value) {
+        if (size == MAX_NUMBER) {
+            throw new RuntimeException("Can't add key: "
+                        + key
+                        + "\nvalue: "
+                        + value
+                        + "Storage is full");
```

Comment:
Good point! Also, you can specify come concrete exception, like `IllegalStateException`.

Also, let's try using `String.format()` here.
```suggestion
            throw new IllegalStateException(String.format("Can't add key: %s, value: %s. Storage is full.", key, value));
```